empress_gui.py

```diff
@@ -4,362 +4,819 @@
+
+            self.mapping_info.destroy()
+
+        # Reset dtl costs so self.compute_reconciliations_btn can be disabled
         self.dup_cost = None
         self.trans_cost = None
         self.loss_cost = None
+        self.dup_input.set(1.00)
+        self.trans_input.set(1.00)
+        self.loss_input.set(1.00)
+        self.compute_reconciliations_btn.configure(state=tk.DISABLED)
+        self.view_solution_space_dropdown.configure(state=tk.DISABLED)
+        self.view_reconciliations_dropdown.configure(state=tk.DISABLED)
+        self.view_pvalue_histogram_btn.configure(state=tk.DISABLED)
+
+        # Reset the rest
+        self.dup_label.destroy()
+        self.dup_entry_box.destroy()
+        self.trans_label.destroy()
+        self.trans_entry_box.destroy()
+        self.loss_label.destroy()
+        self.loss_entry_box.destroy()
+
+        self.recon_MPRs_label.destroy()
+        self.num_MPRs_label.destroy()
+        self.recon_cospeci_label.destroy()
+        self.recon_dup_label.destroy()
+        self.recon_trans_label.destroy()
+        self.recon_loss_label.destroy()
+
+        self.num_cluster_input = tk.IntVar()
+        self.num_cluster_input.set(1)
+        self.num_cluster = None
+
+        self.recon_info_displayed = False
+        
+        if self.cost_space_window is not None and self.cost_space_window.winfo_exists() == 1:
+            self.cost_space_window.destroy()
+
+        if self.entire_space_window is not None and self.entire_space_window.winfo_exists() == 1:
+            self.entire_space_window.destroy()
+        
+        if self.set_num_cluster_window is not None and self.set_num_cluster_window.winfo_exists() == 1:
+            self.set_num_cluster_window.destroy()
+
+        if self.view_solution_space_window is not None and self.view_solution_space_window.winfo_exists() == 1:
+            self.view_solution_space_window.destroy()
+
+        if self.one_MPR_window is not None and self.one_MPR_window.winfo_exists() == 1:
+            self.one_MPR_window.destroy()
+
+        if self.view_reconciliations_window is not None and self.view_reconciliations_window.winfo_exists() == 1:
+            self.view_reconciliations_window.destroy()
+
+        if self.view_pvalue_histogram_window is not None and self.view_pvalue_histogram_window.winfo_exists() == 1:
+            self.view_pvalue_histogram_window.destroy()
+
+    def load_input_files(self, event):
+        """Load in two .nwk files for the host tree and parasite tree, and one .mapping file. Display the number of tips for 
+        the trees and a message to indicate the successful reading of the tips mapping.""" 
+        # Clicking on "Load host tree file" 
+        if self.load_files_var.get() == "Load host tree file":
+            # initialdir is set to be the current working directory
+            input_file = tk.filedialog.askopenfilename(initialdir=os.getcwd(), title="Select a host file")
```

Comment:
Is there a way to only show the files with specified suffixes? I think the newick suffixes are `.nwk`, `.newick`, or `.tree`.

Reply:
Yes. But that function doesn't work for mac... Let me google for other alternatives if possible.

Reply:
It's okay if you cannot do it. Just asking.

Reply:
I think in Mac, there's a way to gray other options out.

Reply:
Others are grey but still chosable?

Reply:
defaultextension='.nwk' didn't work on my mac but it also only allows for one default extension. 
filetypes=[('newick file','*.nwk'), ('mapping files','*.mapping')] is what I tried before and it didn't allow the switching between the two extensions on mac.

Reply:
Those are the relevant features under askopenfilename.

Reply:
By not working on Mac, I meant that I couldn't switch between one type of file name and ('all files', '*').

Reply:
Like the switching menu doesn't work...

Reply:
We can talk about this tonight